release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -95,6 +95,13 @@ For more information about audit log policies, see xref:../security/audit-log-po
 [id="ocp-4-8-networking"]
 === Networking
 
+[id="ocp-4-8-"]
+==== Dual-stack support on bare metal infrastructure with the OVN-Kubernetes cluster network provider
+
+For clusters installed on bare metal infrastructure, the OVN-Kubernetes cluster network provider supports both IPv4 and IPv6 address families.
+
+For clusters upgrading from previous versions of {product-title}, you must convert your cluster to support dual-stack networking. For more information, see xref:../networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc#converting-to-dual-stack[Converting to IPv4/IPv6 dual stack networking].
```

Comment:
So the build isn't failing, but this link isn't resolving for me. I assume this is coming in a related PR, but I wonder why the build isn't mad about it? :thinking: